runtime-light/core/globals/php-script-globals.h

```diff
@@ -8,7 +8,40 @@
 #include "runtime-core/memory-resource/unsynchronized_pool_resource.h"
 #include "runtime-core/runtime-core.h"
 
+namespace PhpServerSuperGlobalIndices {
+
+inline constexpr auto *ARGC = "argc";
```

Comment:
What is the logic behind the constants location? For example `php-script-globals.h` contains `JOB_ID` that doesn't look generic

Reply:
I don't actually see it there